python/cudf/cudf/tests/test_groupby.py

```diff
@@ -1036,9 +1093,8 @@ def test_groupby_nth(n, by):
     )
     gdf = cudf.from_pandas(pdf)
 
-    expect = pdf.groupby(by).nth(n)
-    got = gdf.groupby(by).nth(n)
-
+    expect = pdf.groupby(by, sort=False).nth(n)
+    got = gdf.groupby(by, sort=False).nth(n)
```

Comment:
Doesn't this default to `sort=False` and without sorting the order of output is arbitrary so we can't compare equality between cuDF / Pandas?

Reply:
@kkraus14 Correct, fixed the test to `sort=True` as we get arbitrary outputs.

But given that our sort=True is not consistent with pandas too, should we retain this test? I think yes, but mark it xfail if it fails in CI?